Modules/TPC/src/Clusters.cxx

```diff
@@ -81,22 +80,40 @@ void Clusters::monitorData(o2::framework::ProcessingContext& ctx)
   mQCClusters.analyse();
 
   auto vecPtrNClusters = toVector(mNClustersCanvasVec);
-  o2::tpc::painter::makeSummaryCanvases(mQCClusters.getNClusters(), 300, 0, 0, true, &vecPtrNClusters);
+  o2::tpc::painter::makeSummaryCanvases(mQCClusters.getNClusters(),
```

Comment:
Good point about the copy&paste. I implemented the function.
I could make the CalDet name match the `paramName`. However, due to the "abuse" of the o2 part of the cluster task to become the RawDigits task there would be one confusing name for the `taskParameters` of the RawDigits. `NRawDigitsNBins`, `NRawDigitsXMin` and `NRawDigitsXMax` would need to be called `N_Clusters_NBin`, etc... To avoid this we would need to add a specific CalPad member for `N_RawDigits` to `O2/Detectors/TPC/qc/include/TPCQC/Clusters.h` which is filled in `o2::tpc::qc::Clusters::fillADCValue`. Then the `o2::tpc::qc::Clusters` member of the qc task would always have one empty CalDet member. `N_Clusters` if it is used for raw digits or `N_RawDigits` if it is used for clusters. 
I don't know if we would want that.